.circleci/scripts/config-template.yml

```diff
@@ -276,52 +326,105 @@ jobs: # Each project will have individual jobs for each specific task it has to
                 root: ./
                 paths:
                   - speckle-sharp-ci-tools/Installers
-
+  
   get-ci-tools: # Clones our ci tools and persists them to the workspace
     docker:
       - image: cimg/base:2021.01
     steps:
       - run: # Could not get ssh to work, so using a personal token
           name: Clone
           command: git clone https://$GITHUB_TOKEN@github.com/specklesystems/speckle-sharp-ci-tools.git speckle-sharp-ci-tools
+      - run: # Could not get ssh to work, so using a personal token
```

Comment:
This is actually something we're going to implement very soon! ♥️ I'll remove it before merging it into `main`.

But it's fine for now since we'll need to test the whole release thing on the CI first.